src/PEPSKit.jl

```diff
@@ -112,24 +110,24 @@ module Defaults
     )
     const iterscheme = :fixed
     const gradient_alg = LinSolver(; solver=gradient_linsolver, iterscheme)
-    
+
     # OhMyThreads scheduler defaults
     const scheduler = Ref{Scheduler}()
     function set_scheduler!()
         scheduler[] = Threads.nthreads() == 1 ? SerialScheduler() : DynamicScheduler()
         return nothing
     end
-    function set_scheduler!(scheduler...; kwargs...)
-        scheduler[] = OhMyThreads.Implementation._scheduler_from_userinput(scheduler...; kwargs...)
+    function set_scheduler!(sc; kwargs...)
+        scheduler[] = OhMyThreads.Implementation._scheduler_from_userinput(sc; kwargs...)
```

Comment:
I do think you want the splatting here, as I think it would also work if you supply just `kwargs`, and not a `Symbol` or `Scheduler`. Alternatively, I vaguely recall seeing `NotGiven()` as a default in many of the OhMyThreads functions, so then I would at least supply that as a default argument. (also, `sc` is not really a better variable name than `scheduler` ;)

Reply:
I think actually just supplying kwargs is not allowed in `_scheduler_from_userinput` - I looked at the implementation and it turns out that that case is explicitly caught and throws an error, that's why I removed the splatting :)

Of course we can supply `NotGiven()` by default, then the kwargs are just passed onto the `DynamicScheduler` constructor. I'll do that tomorrow and then we can merge.

The problem with `scheduler` as a name was that it interfered with the `Defaults.scheduler` constant such that it was erroring whenever I called `set_scheduler!`, that's why I changed it.

Reply:
Oh, that obviously makes sense, my bad